plugins/english/scribblehub.ts

```diff
@@ -0,0 +1,286 @@
+            link += '&ge=' + filters.genres.value.exclude.join(',');
+
+        if (filters.genres.value.include?.length || filters.genres.value.exclude?.length)
+            link += '&mgi=' + filters.genre_operator.value;
+
+        if (filters.content_warning.value.include?.length)
+            link += '&cti=' + filters.content_warning.value.include.join(',');
+
+        if (filters.content_warning.value.exclude?.length)
+            link += '&cte=' + filters.content_warning.value.exclude.join(',');
+
+        if (filters.content_warning.value.include?.length ||
+            filters.content_warning.value.exclude?.length)
+            link += '&mct' + filters.content_warning_operator.value;
+
+        link += '&sort=' + filters.sort.value;
+        link += '&order=' + filters.order.value;
+
+        const body = await fetchApi(link).then((result) =>
+            result.text()
+        );
+
+        const loadedCheerio = parseHTML(body);
+        return this.parseNovels(loadedCheerio);
+    }
+
+    async parseNovelAndChapters(novelUrl: string): Promise<Plugin.SourceNovel> {
+        const url = novelUrl;
+        const result = await fetchApi(url);
+        const body = await result.text();
+
+        let loadedCheerio = parseHTML(body);
+
+        const novel: Plugin.SourceNovel = {
+            url,
+            chapters: [],
+        };
+
+        novel.name = loadedCheerio('.fic_title').text();
+
+        novel.cover = loadedCheerio('.fic_image > img').attr('src');
+    
+        novel.summary = loadedCheerio('.wi_fic_desc').text();
+    
+        novel.genres = loadedCheerio('.fic_genre')
+            .map((i, el) => loadedCheerio(el).text())
+            .toArray()
+            .join(",");
+     
+        novel.status = loadedCheerio('.rnd_stats')
+            .next()
+            .text()
+            .includes('Ongoing')
+                ? 'Ongoing'
+                : 'Completed';
+    
+        novel.author = loadedCheerio('.auth_name_fic').text();
+
+        let formData = new FormData();
+        formData.append('action', 'wi_getreleases_pagination');
+        formData.append('pagenum', '-1');
+        formData.append('mypostid', novelUrl.split('/')[4]);
+
+        const data = await fetchApi(
+            `${this.site}wp-admin/admin-ajax.php`,
+            {
+            method: 'POST',
+            body: formData,
+            },
+        );
+        const text = await data.text();
+
+        loadedCheerio = parseHTML(text);
+
+        const chapter: Plugin.ChapterItem[] = [];
+
+        loadedCheerio('.toc_w').each(function () {
+            const chapterName = loadedCheerio(this).find('.toc_a').text();
+            const releaseDate = loadedCheerio(this).find('.fic_date_pub').text();
+            const chapterUrl = loadedCheerio(this).find('a').attr('href');
+
+            if (!chapterUrl) return;
+            chapter.push({
+                name: chapterName,
+                releaseTime: parseMadaraDate(releaseDate),
```

Comment:
parseMadaraDate is not used for this case. I think I will remove it from lib

Reply:
convert it into iso string instead

Reply:
i have copypasta parseMadaraDate into .ts